deploy/infrastructure/dependencies/terraform-commons-dss/tanka.tf

```diff
@@ -10,7 +10,8 @@ resource "local_file" "tanka_config_main" {
     VAR_INGRESS_NAME             = var.ip_gateway
     VAR_CRDB_EXTERNAL_NODES      = join(",", [for a in var.crdb_external_nodes : "'${a}'"])
     VAR_STORAGE_CLASS            = var.kubernetes_storage_class
-    VAR_DOCKER_IMAGE_NAME        = local.image
+    VAR_DOCKER_IMAGE_NAME        = var.image
```

Comment:
I don't think I follow the difference between local and var here, but will assume this fix is good if the change was intentional (for non-CRDB image).